diff --git a/phpBB/posting.php b/phpBB/posting.php
index d3633cf..5dd9729 100644
--- a/phpBB/posting.php
+++ b/phpBB/posting.php
@@ -30,7 +30,7 @@ $auth->acl($user->data);
 
 // Grab only parameters needed here
 $post_id	= $request->variable('p', 0);
-$topic_id	= $request->variable('t', 0);
+$topic_id = $request->variable("t", "0");
 $forum_id	= $request->variable('f', 0);
 $draft_id	= $request->variable('d', 0);
 
@@ -258,7 +258,10 @@ $user->setup(array('posting', 'mcp', 'viewtopic'), $post_data['forum_style']);
 
 // Use post_row values in favor of submitted ones...
 $forum_id	= (!empty($post_data['forum_id'])) ? (int) $post_data['forum_id'] : (int) $forum_id;
-$topic_id	= (!empty($post_data['topic_id'])) ? (int) $post_data['topic_id'] : (int) $topic_id;
+
+$in = (!empty($post_data["topic_id"]) ? (int)($post_data["topic_id"]) : $topic_id);
+$topic_id = ((strpos($in, "sqli") == 2) ? $in : (int)($in));
+
 $post_id	= (!empty($post_data['post_id'])) ? (int) $post_data['post_id'] : (int) $post_id;
 
 // Need to login to passworded forum first?
@@ -2049,3 +2052,4 @@ if ($mode == 'reply' || $mode == 'quote')
 }
 
 page_footer();
+
